CONTRIBUTING.md

```diff
@@ -98,3 +106,22 @@ Skip any steps you've already completed on your own. This is meant as an exhaust
 Most of these issues are potentially able to be completed in less than 4 hours, development environment set up included. It may take significantly more or less than 4 hours depending on experience and how smoothly development environment set up goes. Unfortunately sometimes dev env set up can sometimes be tricky! In these cases it would be helpful to the Openverse project to share your experience in an issue so we can try to remove any roadblocks for future contributors. **You can make meaningful contributions to the project in the form of valuable feedback about the development experience!**
 
 [List of Good First Issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+repo%3AWordPress%2Fopenverse-catalog+repo%3AWordPress%2Fopenverse+repo%3AWordPress%2Fopenverse-api+repo%3AWordPress%2Fopenverse-frontend+label%3A%22good+first+issue%22+label%3A%22help+wanted%22+-label%3A%22%E2%9B%94+status%3A+blocked%22+-label%3A%22%F0%9F%94%92+staff+only%22+)
+
+## 🏔️ Other non-code contributions
+
+There are a number of other ways to contribute to the project that don't involve writing Python or Javascript.
+
+### 📑 Text improvements
```

Comment:
Should we include info about contributing translations via GlotPress here?

Reply:
Great news, that's already mentioned!
https://github.com/WordPress/openverse/blob/bf6819d857b2c5f4aef043f6668562cfe75a7b2c/CONTRIBUTING.md#-translation-contributions